core-web/libs/dotcms-scss/angular/dotcms-theme/components/_table.scss

```diff
@@ -8,25 +13,278 @@ p-table {
+                }
+            }
+        }
+
+        & .p-datatable-tbody > tr {
+            background: $white;
+            color: $black;
+            height: auto;
+            transition: none;
+            outline: 0 none;
+            cursor: pointer;
+            overflow-wrap: anywhere;
+
+            &:hover {
                 background-color: $color-palette-primary-200;
             }
+
+            & > td {
+                text-align: left;
+                border: 1px solid $color-palette-gray-300;
+                border-width: 0 0 1px 0;
+                padding: 0.625rem $spacing-1;
+                font-size: $font-size-smd;
+
+                &:first-child {
+                    padding-left: $spacing-3;
+                }
+
+                &:last-child {
+                    padding-right: $spacing-3;
+                }
+
+                & .p-row-toggler,
+                & .p-row-editor-init,
+                & .p-row-editor-save,
+                & .p-row-editor-cancel {
+                    width: $spacing-5;
+                    height: $spacing-5;
+                    color: $black;
+                    border: 0 none;
+                    background: transparent;
+                    border-radius: 50%;
+                    transition: background-color 0.2s, color 0.2s, box-shadow 0.2s;
+
+                    &:enabled:hover {
+                        color: $text-color-hover;
+                        border-color: transparent;
+                        background: $color-palette-primary-100;
+                    }
+
+                    &:focus {
+                        outline: 0 none;
+                        outline-offset: 0;
+                        box-shadow: none;
+                    }
+                }
+
+                & .p-row-editor-save {
+                    margin-right: $spacing-1;
+                }
+
+                .pi.pi-ellipsis-v {
+                    color: $color-palette-primary-500;
+                }
+            }
+
+            &.p-highlight {
+                background: $bg-highlight;
+                color: $text-color-highlight;
+            }
+
+            &.disabled-row {
+                background: $color-palette-gray-200;
+                color: $color-palette-gray-700;
+                cursor: default;
+            }
+
+            &.p-datatable-dragpoint-top > td {
+                box-shadow: inset 0 2px 0 0 $color-palette-primary-100;
+            }
+
+            &.p-datatable-dragpoint-bottom > td {
+                box-shadow: inset 0 -2px 0 0 $color-palette-primary-100;
+            }
+
+            &:not(.p-highlight):hover {
+                background: $color-palette-primary-100;
+                color: $text-color-hover;
+                transition: background-color ease-in $basic-speed;
+            }
+
+            &.disabled-row:hover {
+                background: $color-palette-gray-200;
+                color: $color-palette-gray-700;
```

Comment:
Disable row will have hover?

Reply:
to overwrite the default hover + preserve the theme structure as possible.